src/modules/universal-discovery/components/create/create.choose.content.type.component.js

```diff
@@ -7,19 +7,50 @@ export default class ChooseContentTypeComponent extends Component {
     constructor(props) {
         super(props);
 
+        let selected = {};
+        const isForcedContentType = props.allowedContentTypes.length === 1;
```

Comment:
The forced content type should have a flag. Checking if there is only one allowed content type doesn't that it is a forced one.

Reply:
This is what we assume: one allowed means forced. I can pass additional prop to this component but the check would be the same `length === 1`.